apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -200,6 +201,19 @@ const PaymentsTableContent: React.FC<IPaymentsTableProps> = ({ billingContext, c
         }
     }, [])
 
+    /**
+    * set default filters only when filters are empty and reset button hasn't been pressed
+    **/
+    useEffect(()=>{
+        if (hasFilters){
+            filtersAreReset = false
```

Comment:
If you don't want to trigger rerender here you may use react ref as a value storage